api/files/serializers.py

```diff
@@ -213,7 +213,10 @@ class BaseFileSerializer(JSONAPISerializer):
         read_only=True, help_text='The Unix-style path of this object relative to the provider root',
     )
     last_touched = VersionedDateTimeField(read_only=True, help_text='The last time this file had information fetched about it via the OSF')
-    date_modified = ser.SerializerMethodField(read_only=True, help_text='Timestamp when the file was last modified')
+    date_modified = VersionedDateTimeField(
+        read_only=True,
```

Comment:
Add `required=False` and `null=True` here in case we missed spots where this serializer being used